### PR TITLE
Update replicaset.md

### PR DESCRIPTION
**Pull request by gouthamchilakala**
Added the lines 293-296

Added the command to notify users on how to edit the POD YAML spec to change the Label name.

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For pull requests on Chinese localization, set Base Branch to release-1.14
>
> For pull requests on Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by zacharysarah** [comment]
@kubernetes/sig-apps-pr-reviews 👋 Would you please review this PR for accuracy? Thanks in advance!

File: content/en/docs/concepts/workloads/controllers/replicaset.md [hunk 1]
```diff
@@ -290,6 +290,11 @@ You can remove Pods from a ReplicaSet by changing their labels. This technique m
 from service for debugging, data recovery, etc. Pods that are removed in this way will be replaced automatically (
 assuming that the number of replicas is not also changed).
 
+Change the label name in the POD YAML definition:
+```shell
+kubectl edit POD <POD NAME>
```
Comment (xiangpengzhao) [inline content/en/docs/concepts/workloads/controllers/replicaset.md]: Should the first `POD` be `pod`? Also, in line 198 of this file, `kubectl get Pods` should be `kubectl get pods`

File: content/en/docs/concepts/workloads/controllers/replicaset.md [hunk 2]
```diff
@@ -290,6 +290,11 @@ You can remove Pods from a ReplicaSet by changing their labels. This technique m
 from service for debugging, data recovery, etc. Pods that are removed in this way will be replaced automatically (
 assuming that the number of replicas is not also changed).
 
+Change the label name in the POD YAML definition:
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/replicaset.md]:
```suggestion
Edit a Pod's manifest (to change its labels):
```
How about this?

**Comment by zacharysarah** [comment]
I've pinged SIG Apps for a review on [Slack](https://kubernetes.slack.com/archives/C18NZM5K9/p1570572670024800).

File: content/en/docs/concepts/workloads/controllers/replicaset.md (hunk 1 above)
Comment (zparnold) [inline content/en/docs/concepts/workloads/controllers/replicaset.md]:
```suggestion
kubectl edit pod <POD NAME>
```

File: content/en/docs/concepts/workloads/controllers/replicaset.md (hunk 2 above)
Comment (sftim) [reply content/en/docs/concepts/workloads/controllers/replicaset.md]: @gouthamchilakala what do you think?

**Comment by tengqm** [comment]
Closing because it has been stale for 3+ months. Feel free to reopen.

**Pull request closed without merging**